ext/NNlibCUDACUDNNExt/conv.jl

```diff
@@ -67,6 +68,23 @@ function conv!(y::DenseCuArray{T}, x::DenseCuArray{T}, w::DenseCuArray{T}, cdims
     cudnnConvolutionForward!(y, w, x, d; alpha, beta, z=y)
 end
 
+function conv!(y::DenseCuArray{T}, x::DenseCuArray{T}, w::DenseCuArray{T}, cdims::DenseConvDims;
```

Comment:
can you write a comment explaining the trick or add some references?

Reply:
np, added a comment above `conv!` explaining.